### PR TITLE
FEA-84 - add favorites list component to the main view

### PR DESCRIPTION
**Pull request by lukasz-konatowski-cc**
reused previous NearestCitiesWeatherWIdget container in FavoritesWeatherWidget component

### CODE REVIEW

File: src/pages/Home/components/FavoritesCitiesWeatherWidget/FavoritesCitiesWeatherWidget.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+const Container = styled.div``;
```
Comment (bednarskimichal) [inline src/pages/Home/components/FavoritesCitiesWeatherWidget/FavoritesCitiesWeatherWidget.styled.tsx]: leftover?

Comment (lukasz-konatowski-cc) [reply src/pages/Home/components/FavoritesCitiesWeatherWidget/FavoritesCitiesWeatherWidget.styled.tsx]: @bednarskimichal  Yes it is, thank you :)

File: src/pages/City/components/NearestCitiesWeatherWidget/NearestCitiesWeatherWidget.styled.tsx [hunk 2]
```diff
@@ -1,18 +1,5 @@
...
-  align-items: center;
-
-  @media screen and (max-width: ${({ theme }) => theme.widths.tablet}) {
-    & {
-      order: 2;
-    }
-  }
-`;
-
 const ShortcutsWrapper = styled.div`
```
Comment (AgataLudwiczynska-CC) [inline src/pages/City/components/NearestCitiesWeatherWidget/NearestCitiesWeatherWidget.styled.tsx:15]: **question:** ‏When you added this `ShortcutWrapper` to `CitiesWeatherWidget`, shouldn't you have removed it here?

Comment (lukasz-konatowski-cc) [reply src/pages/City/components/NearestCitiesWeatherWidget/NearestCitiesWeatherWidget.styled.tsx:15]: That's true, indeed :) 
Thanks :)

Comment (lukasz-konatowski-cc) [reply src/pages/City/components/NearestCitiesWeatherWidget/NearestCitiesWeatherWidget.styled.tsx:15]: Done 👍

**Review by AgataLudwiczynska-CC — APPROVED** [review]

File: src/components/CitiesWeatherWidget/components/Title.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { Name } from './components.styled';
+
+interface Props {
+  name: string;
+}
+
+const Title = ({ name }: Props) => {
+  return <Name>{name}</Name>;
+};
```
Comment (JakubLukaszCC) [inline src/components/CitiesWeatherWidget/components/Title.tsx]: **suggestion** i think it is unnecessary to create whole component just to add styling. I would suggest to just use `Name` styled component in CitiesWeatherWidget like this `CitiesWeatherWidget.Title = Name;`

**Review by adrian-konopnicki-CC — APPROVED** [review]

File: src/components/CitiesWeatherWidget/components/Title.tsx (hunk 3 above)
Comment (JakubLukaszCC) [reply src/components/CitiesWeatherWidget/components/Title.tsx]: also the same can be done to `ShortcutWrapper`

Comment (lukasz-konatowski-cc) [reply src/components/CitiesWeatherWidget/components/Title.tsx]: Good point :)

**Review by JakubLukaszCC — APPROVED** [review]

**Review by bednarskimichal — APPROVED** [review]

**Pull request merged**